app/dashboard/views.py

```diff
@@ -775,74 +721,104 @@ def toolbox(request):
              "description": '''CodeSponsor sustains open source
                         by connecting sponsors with open source projects.''',
              "link": "https://codesponsor.io",
+             'link_copy': 'Try It',
              "active": "false",
              'stat_graph': 'codesponsor',
         }
         ]
       }, {
-          "title": "The Powertools",
+          "title": "Advanced",
           "description": "Take your OSS game to the next level!",
           "tools": [{
-              "name": "Browser Extension",
+              "name": "Chrome Browser Extension",
               "img": static("v2/images/tools/browser_extension.png"),
               "description": '''Browse Gitcoin where you already work.
                     On Github''',
               "link": reverse("browser_extension"),
+              'link_copy': 'Try It',
               "active": "false",
               'stat_graph': 'browser_ext_chrome',
-          },
-          {
-              "name": "iOS app",
-              "img": static("v2/images/tools/iOS.png"),
-              "description": '''Gitcoin has an iOS app in alpha. Install it to
-                browse funded work on-the-go.''',
-              "link": reverse("ios"),
+          },{
+              "name": "gitcoinbot",
+              "img": static("v2/images/helmet.png"),
```

Comment:
is this missing the relative `/` in the beginning or is it add else where? If the former should it then be: `static("/v2/images/helmet.png")`

Reply:
The slash isn't required here.